mesh/pncmesh.cpp

```diff
@@ -893,7 +893,15 @@ void ParNCMesh::GetFaceNeighbors(ParMesh &pmesh)
    Array<Element*> fnbr;
    Array<Connection> send_elems;
 
-   int bound = shared.conforming.Size() + shared.slaves.Size();
+   // Counts the number of slave faces of a master. This may be larger than the
+   // number of shared slaves if there exist degenerate slave-faces from face-edge constraints.
+   auto count_slaves = [&](int i, const Master& x)
+   {
+      return i + (x.slaves_end - x.slaves_begin);
+   };
+
+   const int bound = shared.conforming.Size() + std::accumulate(
```

Comment:
AppVeyor is failing on `std::accumulate`. I suspect we need to explicitly include `<numeric>` here.

Reply:
Sorry I missed that, I admit with all the unrelated CI failures I had stopped looking. Fixed in https://github.com/mfem/mfem/pull/3587/commits/58db0c0d65ed78228fce4980c07fc084f0deb88c